.gitignore

```diff
@@ -110,3 +110,6 @@ website/public/
 website/resources/
 website/node_modules/
 website/tech-doc-hugo
+
+# Python cache (Ansible molecule)
+test/ansible/plugins/filter/__pycache__/
```

Comment:
let's add the same for the sample just to ensure that if we use it to test some change locally we will not commit it as well.